modules/installation-configuration-parameters.adoc

```diff
@@ -1882,6 +1882,140 @@ with an Azure cluster.
 ====
 endif::azure[]
 
+
+ifdef::agent[]
+[id="installation-configuration-parameters-additional-bare_{context}"]
+== Additional bare metal configuration parameters for the Agent-based Installer
+
+Additional bare metal installation configuration parameters for the Agent-based Installer are described in the following table:
```

Comment:
Should indicate that these fields are not used during the initial provisioning of the cluster - since that is done by the assisted-installer - but will be available to use after the the cluster has been installed. Configuring these fields at install time save the user extra step of setting them in Day 2.